be/src/runtime/descriptors.h

```diff
@@ -204,48 +203,37 @@ class HdfsTableDescriptor : public TableDescriptor {
         return Status::OK();
     }
 
-private:
+protected:
     std::string _hdfs_base_dir;
     std::vector<TColumn> _columns;
     std::vector<TColumn> _partition_columns;
     std::map<int64_t, HdfsPartitionDescriptor*> _partition_id_to_desc_map;
+    std::string _table_location;
 };
 
-class IcebergTableDescriptor : public TableDescriptor {
+class HdfsTableDescriptor : public HdfsBaseTableDescriptor {
```

Comment:
HdfsTable -> HiveTable

Reply:
OK.

Reply:
I guess it's not good idea. Because there are a lof names are using `HDFS table`. For consistency, we'd better not to change that.